libkbfs/test_common.go

```diff
@@ -79,6 +80,32 @@ func MakeTestBlockServerOrBust(t logger.TestLogBackend,
 	}
 }
 
+// TODO: Put the below code (also duplicated in kbfs-server) somewhere
+// in rpc and use that instead.
+
+type testLogger interface {
+	Logf(format string, args ...interface{})
+}
+
+type testLogOutput struct {
+	t testLogger
+}
+
+func (t testLogOutput) log(ch string, fmts string, args []interface{}) {
+	fmts = fmt.Sprintf("[%s] %s", ch, fmts)
+	t.t.Logf(fmts, args...)
+}
+
+func (t testLogOutput) Info(fmt string, args ...interface{})    { t.log("I", fmt, args) }
+func (t testLogOutput) Error(fmt string, args ...interface{})   { t.log("E", fmt, args) }
+func (t testLogOutput) Debug(fmt string, args ...interface{})   { t.log("D", fmt, args) }
+func (t testLogOutput) Warning(fmt string, args ...interface{}) { t.log("W", fmt, args) }
+func (t testLogOutput) Profile(fmt string, args ...interface{}) { t.log("P", fmt, args) }
+
+func newTestRPCLogFactory(t testLogger) rpc.LogFactory {
+	return rpc.NewSimpleLogFactory(testLogOutput{t}, nil)
```

Comment:
This'll cause all RPCs to be logged, but only if tests are run to point to an external server